osu.Game/Screens/Backgrounds/BackgroundModeBeatmap.cs

```diff
@@ -22,17 +22,19 @@ public WorkingBeatmap Beatmap
             {
                 return beatmap;
             }
-
             set
             {
-                if (beatmap == value)
+                if (beatmap == value && beatmap != null)
                     return;
-
                 beatmap = value;
 
                 Schedule(() =>
                 {
-                    Background newBackground = new BeatmapBackground(beatmap);
+                    Background newBackground;
+                    if (beatmap == null)

```

Comment:
:-1: I would find that very annoying. A whole class of problems is really just 2 or 3 problems, which have all been addressed as of this PR.